site/pages/admin/user-management.page.tsx

```diff
@@ -141,6 +158,10 @@ const UserManagement = ({ userList, csrfToken }: UserManagementPageProps): React
                                 name: "group",
                                 value: userToResendInvite.userGroup,
                             },
+                            {
+                                name: "orgId",
```

Comment:
Do we need to pass through the orgId on this page? We should be grabbing it from the session in the case the user is an org admin

Reply:
removed